src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -539,20 +540,24 @@ private CompilationTracker GetCompilationTracker(ProjectId projectId)
 
         private Solution AddProject(ProjectId projectId, ProjectState projectState)
```

Comment:
Good catch--I originally had this in ForkProject and intended to move it into AddDocument and RemoveDocument.